CanvasColorSpaceProposal.md

```diff
@@ -164,40 +159,55 @@ Note: An alternative approach that was considered was to augment the probablySup
 
 #### ImageBitmap
 
-TODO(ccameron): Review this
-ImageBitmap objects are augmented to have an internal color space attribute of type CanvasColorSpace and an internal pixelFormat attribute of type CanvasPixelFormat. The colorSpaceConversion creation attribute also accepts enum values that correspond to CanvasColorSpace values. Specifying a CanvasColorSpace value results in a conversion of the image to the specified color space.
+ImageBitmap objects (unless created with ``colorSpaceConversion="none"``) should keep track of their internal color space, and should store their contents at highest fidelity possible, subject to implementation limitations.
 
 #### ImageData
 
-TODO(ccameron): Review this
-
-IDL
+Add the following types to be used by `ImageData`.
 <pre>
-
-enum ImageDataStorageType {
-  "uint8", // default
-  "uint16",
+enum ImageDataEncoding {
+  "unorm8", // default
+  "unorm16",
```

Comment:
What's the motivation behind this change? It introduces a new concept which doesn't align with the Typed Array specification - these will be accessed with Uint8Array and Uint16Array. I would prefer to leave these as "uint8" and "uint16", and define elsewhere that these are treated as unsigned normalized values.

Reply:
I'm agnostic. Theoretically, one could request an actual uint (as in, integer texture) encoding, but that's probably just a footgun. I've changed it back.